django/db/backends/base/validation.py

```diff
@@ -1,10 +1,30 @@
+from django.core import checks
+
+
 class BaseDatabaseValidation:
     """Encapsulate backend-specific validation."""
     def __init__(self, connection):
         self.connection = connection
 
     def check(self, **kwargs):
-        return []
+        return self._check_database_version()
+
+    def _check_database_version(self):
+        errors = []
+        if not self.connection.is_database_version_supported():
+            db_version_str = '.'.join(map(str, self.connection.get_database_version()))
```

Comment:
I think we can remove the "_str" suffix from these variables. They were only needed to avoid a naming collision in a previous implementation.